roles/openshift_register_nodes/library/kubernetes_register_node.py

```diff
@@ -255,7 +255,7 @@ def exists(self):
         return False
 
     def create(self):
-        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', 'node', '-f', '-']
+        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', '-f', '-']
```

Comment:
Would it be clearer to always use 'openshift kube' vs 'osc' and then 'openshift +verb+' other places?